app/uk/gov/hmrc/cataloguefrontend/platforminitiatives/PlatformInitiativesListPage.scala.html

```diff
@@ -120,12 +135,23 @@ <h3> @initiative.initiativeName </h3><br \>
                                 </div>
                             }
                             case DisplayType.Progress => {
-                                <div class="initiativeName" style="width:500px;">
+                                <div class=" initiativeName" style="width:500px;">
                                     <h3> @initiative.initiativeName </h3>
-                                    <progress class="progress" value="@initiative.currentProgress"
-                                              max="@initiative.targetProgress">
+                                    @defining(initiative.currentProgress.toFloat/initiative.targetProgress.toFloat * 100) { percent =>
```

Comment:
add `def percent: Int` to Initiative? (This is where a value object for the `Progress` would be useful)